code/modules/mob/living/simple_animal/hostile/deer.dm

```diff
@@ -0,0 +1,147 @@
+/*
+TODO -
```

Comment:
I like leaving the TODO list.

Lets me and others know what the hell's going on in this pile of pasta